packages/driver/src/cross-origin/cypress.ts

```diff
@@ -165,21 +165,18 @@ const attachToWindow = (autWindow: Window) => {
 
   const cy = Cypress.cy
 
+  // this communicates to the injection code that Cypress is now available so
+  // it can safely subscribe to Cypress events, etc
+  // @ts-ignore
+  autWindow.__attachToCypress(Cypress)
```

Comment:
lol, I have basically this same thing in my PR: https://github.com/cypress-io/cypress/pull/24243

Reply:
Great minds and all 🧠